### ISSUE TITLE
rangeOfString: conversion

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#120]
The `rangeOfString:` family of methods in ObjC are converted to the same family in Swift, but:

- in Swift, they return `nil` if nothing is found
- in ObjC, they return `NSRange` with `.location = NSNotFound`

Example:

```
[[app bundleIdentifier] rangeOfString:@"obdev" options:NSCaseInsensitiveSearch].location != NSNotFound
```

This should be in Swift:

```
app.bundleIdentifier.rangeOfString("obdev", options: .CaseInsensitiveSearch) != nil
```

**Comment by drkameleon** [comment]
Well, this one is a bit tricky as well...

Basically, think of how the parser sees things: 

- This is an expression : `[[app bundleIdentifier] rangeOfString:@"obdev" options:NSCaseInsensitiveSearch].location` (which we can then "break down" to its part: a method selector and a property)
- This is another expression : `nil`
- And the whole of it (A != B) is another expression.

So, as you can see it can get rather complicated, given that we'll have to know the return type of the first expression to make any sort of such assumptions.

**The question is:** does `NSNotFound` still apply? (in any other case). If not, then we can safely replace all `NSNotFound`s with `nil`. If, however, it's not something universal, or if it is a `rangeOfString:` specific thing, then I'm afraid this goes under the "very difficult to implement" category.

**Comment by drkameleon** [comment]
Also... given what I'm seeing in very questions @ SO, would a conversion to `Foundation.NSNotFound` make sense?

**Comment by charlieMonroe** [comment]
I would personally replace all occurrences of `.location == NSNotFound` with ` == nil` (and the same with `!=`)...

**Comment by drkameleon** [comment]
Fixed as of the upcoming 2.1 - following your suggestion.

--

**Example:**
```objective-c
@implementation aClass

-(void)someFunc {

    if ([[app bundleIdentifier] rangeOfString:@"obdev" options:NSCaseInsensitiveSearch].location!= NSNotFound) {
        
    }
}

@end
```

**Output:**
```swift
class aClass {

	func someFunc() {
		if app.bundleIdentifier().rangeOfString("obdev", options: NSCaseInsensitiveSearch) != nil {
		
		}
	
	}
	
}
```
